pkgs/tools/networking/snabb/default.nix

```diff
@@ -0,0 +1,32 @@
+{stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  name = "snabb-2015.10";
+
+  src = fetchurl {
```

Comment:
This should use `fetchFromGitHib`. `fetchurl` hashes tend to break with GitHub (since the tarballs are generated dynamically).